src/components/pages/itemDetail/itemDetail.jsx

```diff
@@ -0,0 +1,414 @@
+            }
+        }
+
+        if (productId) {
+            fetchData();
+        }
+        
+    }, [productId]);
+
+    if (!item) return <div>로딩 중...</div>;
+
+    return(
+        <div css={pageWrapper}>
+            <div css={itemDetailWrapper}>
+                <div css={imgSection}>
+                    <img src={item.images || defaultImage}
+                        onError={(e)=> {
+                            e.target.onerror = null;
+                            e.target.src = defaultImage;
+                        }}    
+                    />
+                </div>
+                <div css={itemDetail}>
+                    <div css={itemHeader}>
+                        <div css={titleHeader}>
+                            <h3>{item.name}</h3>
+                            <img src={kebabIcon} css={editIconStyle} />
+                        </div>
+                        <h2>{item.price}원</h2>
+                    </div>
+                    <div css={itemDescription}>
+                        <h3>상품소개</h3>
+                        <h4>{item.description}</h4>
+                    </div>
+                    <div css={itemTag}>
+                        <h3>상품 태그</h3>  
+                            <div css={tagWrapper}>
+                                {item.tags.map((tag)=> {
+                                    return(
+                                        <div key={item.id} css={tagBox}>
+                                            <span>#{tag}</span>
+                                        </div>
+                                    )
+                                })}
+                            </div>
+                    </div>
+                    <div css={itemUserInfo}>
+                        <div css={userImg}>
+                            <img src={userImage} />
+                        </div>
+                        <div css={userText}>
+                            <h4>{item.ownerNickname}</h4>
+                            <p>{new Date(item.createdAt).toISOString().slice(0, 10)}</p>
+                        </div>
+                        <div css={likeSection}>
+                            <button>💛</button>
+                            {item.favoriteCount}
+                        </div>
+                    </div>
+                </div>
+            
+            </div>
+
+            <div css={commentSection}>
+                <div css={makeComment}>
+                    <label htmlFor="makecomment">문의하기</label>
+                    <textarea 
+                        id="makecomment" 
+                        placeholder="개인정보를 공유 및 요청하거나, 명예 훼손, 무단 광고, 불법 정보 유포시 모니터링 후 삭제될 수 있으며, 이에 대한 민형사상 책임은 게시자에게 있습니다."
+                        value={commentTextarea}
+                        onChange={handleChange}
+                        />
+                    <button type="button" css={[commentButtonStyle, isButtonActive && activeButtonStyle]} >등록</button>
```

Comment:
❗️ 수정요청
지금의 경우 스타일만 disabled처럼 보이게 적용이 된 것이고 실제로는 클릭이 가능하게 처리되어 있습니다.
불필요한 렌더링을 줄이기 위해 commentButtonStyle 내부에 :disabled 가상 클래스로 스타일링해주세요~

```suggestion
                    <button type="button" disabled={isButtonActive} css={commentButtonStyle}>등록</button>
```